src/screens/NationCreateContainer/index.js

```diff
@@ -67,26 +74,33 @@ class NationCreateContainer extends Component {
       }]);
   };
 
-  _deleteForm = () => {
+  deleteForm = () => {
     alert('deleteForm', [
       {
         name: 'cancel',
         style: 'cancel',
       }, {
         name: 'delete',
         style: 'destructive',
-        onPress: () => this.props.onDeleteNationDraft(this.props.initialNation.id, () => {
-          if (this.props.latestError) {
-            errorAlert(this.props.latestError);
-            return;
-          }
-
-          this.props.navigator.dismissModal();
-        }),
+        onPress: () => {
+          if (this.props.initialNation === null) return;
+          if (typeof this.props.initialNation.id !== 'number') return;
```

Comment:
This check is unnecessary here (was required earlier, but we fix that by change nation types). Please remove it.

Reply:
You're right. Done

Reply:
Changed second line to: 
`if (this.props.initialNation.id == null) return;`